docs/content/quickstart/quick-start-for-CDC-3.0-pipeline.md

```diff
@@ -0,0 +1,44 @@
+# Flink CDC 3.0 QuickStart
+
+> User scenario:
+When building their own real-time streaming lakehouse using Doris (Starrocks, Paimon, and Iceberg), users need to continuously synchronize data, including snapshot data and incremental data, from multiple business tables in MySQL database to Doris for creating the ODS layer. 
+
+## Install Flink CDC and required dependencies
+
+Download and extract the flink-cdc-3.0.tar file to a local directory.
```

Comment:
I wonder how to download flink-cdc-3.0.tar and Pipeline Connector JAR, Could you please give more details or links. This quickstart use mysql to doris, how can user like me download the current jar?

Reply:
At present, there is no link available, and this part will be supplemented after the corresponding function is completed.